client/src/OnDeviceComponent.ts

```diff
@@ -311,6 +311,45 @@ export class OnDeviceComponent {
 		} & ODC.ReturnTimeTaken;
 	}
 
+	public async startResponsivenessTesting(args: ODC.StartResponsivenessTestingArgs = {}, options: ODC.RequestOptions = {}) {
+		const result = await this.sendRequest('startResponsivenessTesting', args, options);
+		return result.json as ODC.ReturnTimeTaken;
+	}
+
+	public async getResponsivenessTestingData(args = {}, options: ODC.RequestOptions = {}) {
+		const result = await this.sendRequest('getResponsivenessTestingData', args, options);
+		return result.json as ODC.ReturnTimeTaken & {
+			periods: {
+				/** Duration of this period in milliseconds */
+				duration: number;
+
+				/** What percent of the time during the period the render thread was responsive */
+				percent: number;
```

Comment:
It might be good to clarify in a comment what this value is. Is it `0.0` to `1.0` or is it `0`-`100`? The term "percent" gets pretty loaded.